src/User32.Desktop/User32+DISPLAYCONFIG_SOURCE_MODE_INFO.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="DISPLAYCONFIG_SOURCE_MODE_INFO"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        [StructLayout(LayoutKind.Sequential)]
+        public struct DISPLAYCONFIG_SOURCE_MODE_INFO
+        {
+            public uint bitvector1;
```

Comment:
I'm not a fan of changing original names but theses ones are so lame that they may be worth it 👍